create_completion.py

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env python3
 
 import openai
```

Comment:
Might as well drop this import (the `import openai` one).

Reply:
Thanks! I did so.